sky/packages/sky/lib/src/fn3/editable_text.dart

```diff
@@ -207,29 +195,67 @@ class EditableTextState extends State<EditableText> {
     else if (!config.focused && _cursorTimer != null)
       _stopCursorTimer();
 
-    final EditableString value = config.value;
-    final TextStyle style = config.style;
+    return new _EditableTextWidget(
+      value: config.value,
+      style: config.style,
+      cursorColor: config.cursorColor,
+      showCursor: _showCursor,
+      onContentSizeChanged: config.onContentSizeChanged,
+      scrollOffset: config.scrollOffset);
+  }
+}
+
+class _EditableTextWidget extends LeafRenderObjectWidget {
+  _EditableTextWidget({
+    Key key,
+    this.value,
+    this.style,
+    this.cursorColor,
+    this.showCursor,
+    this.onContentSizeChanged,
+    this.scrollOffset
+  }) : super(key: key);
 
-    Widget text;
+  final EditableString value;
+  final TextStyle style;
+  final Color cursorColor;
+  final bool showCursor;
+  final SizeChangedCallback onContentSizeChanged;
+  final Offset scrollOffset;
+
+  RenderEditableParagraph createRenderObject() => new RenderEditableParagraph(
```

Comment:
style nits:

- only use => if the expression fits on the same line. Otherwise, use a block with return.
- If you have a newline after a (, {, or [, you should have a newline before the corresponding ), }, or ]. (In fact in general, whitespace around open/close punctuation should be symmetric.)
- Indent two spaces from the previous indent.